recipes/log4cpp/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./configure --prefix="${PREFIX}"
+make -j${CPU_COUNT}
+make install
```

Comment:
Is there a `make check` you can run?

Reply:
There is a ``make check``, I was using that but I removed. However I will add that again.

Thanks! :)